bigtop-manager-server/src/main/resources/stacks/bigtop/3.3.0/services/solr/metainfo.xml

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0"?>
+<!--
+  ~ Licensed to the Apache Software Foundation (ASF) under one
+  ~ or more contributor license agreements.  See the NOTICE file
+  ~ distributed with this work for additional information
+  ~ regarding copyright ownership.  The ASF licenses this file
+  ~ to you under the Apache License, Version 2.0 (the
+  ~ "License"); you may not use this file except in compliance
+  ~ with the License.  You may obtain a copy of the License at
+  ~
+  ~    https://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing,
+  ~ software distributed under the License is distributed on an
+  ~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  ~ KIND, either express or implied.  See the License for the
+  ~ specific language governing permissions and limitations
+  ~ under the License.
+-->
+<metainfo>
+    <schema-version>2.0</schema-version>
+    <service>
+            <name>solr</name>
+            <display-name>Solr</display-name>
+            <desc>
+                Solr is the popular, blazing-fast, open source enterprise search platform built on Apache Lucene.
+            </desc>
+            <version>8.11.2</version>
+            <user>solr</user>
+            <group>solr</group>
+            <components>
+                <component>
+                    <name>SOLR</name>
```

Comment:
Make sure component name is lowercase and similar to display name, it should be `solr_instance` here